main.nf

```diff
@@ -843,6 +854,7 @@ process preseq {
  * STEP 6 Mark duplicates
  */
 process markDuplicates {
+    label 'low_memory'
```

Comment:
Is MarkDuplicates really low Memory?

Reply:
I just took this based on what we had previously: https://github.com/nf-core/rnaseq/blob/3b68919c56653981a3bc25041304ee6c47f1c6ce/conf/base.config#L70-L72

Reply:
Aah I see :-)